satellite/metainfo/metainfo_test.go

```diff
@@ -453,29 +453,29 @@ func TestValueAttributeInfo(t *testing.T) {
 		require.NoError(t, err)
 
 		keyInfo := pb.ValueAttributionRequest{
-			PartnerId:  []byte("PartnerID"),
-			BucketName: []byte("myBucketName"),
+			PartnerId:  []byte("I am the new PartnerID"),
+			BucketName: []byte("myProjectID  +  myBucketName"),
 		}
 
 		{
 			// bucket with no items
-			err = metainfoClient.ValueAttributeInfo(ctx, "myBucket", "", -1, string(keyInfo.PartnerId))
+			err = metainfoClient.ValueAttributeInfo(ctx, string(keyInfo.GetBucketName()), "", -1, string(keyInfo.PartnerId))
 			require.NoError(t, err)
 
 			// no bucket exists
-			err = metainfoClient.ValueAttributeInfo(ctx, "myBucket1", "", -1, string(keyInfo.PartnerId))
+			err = metainfoClient.ValueAttributeInfo(ctx, string(keyInfo.GetBucketName())+"1", "", -1, string(keyInfo.PartnerId))
 			require.NoError(t, err)
 		}
 		{
 			expectedData := make([]byte, 1*memory.MiB)
 			_, err = rand.Read(expectedData)
 			assert.NoError(t, err)
 
-			err = planet.Uplinks[0].Upload(ctx, planet.Satellites[0], "myBucket", "path", expectedData)
+			err = planet.Uplinks[0].Upload(ctx, planet.Satellites[0], string(keyInfo.GetBucketName()), "path", expectedData)
 			assert.NoError(t, err)
```

Comment:
use `require` unless `assert` is necessary

Reply:
code refactored and test cases updated.